README.md

```diff
@@ -125,7 +137,7 @@ Write code to do the following:
 
 Write your code here:
 ```ruby
-# code here
+# I'm not sure what I need to do here. Will study up on this asap.
 ```
```

Comment:
The question is asking you to write out ActiveRecord methods that correspond to the actions described in the list. So the first one that asks for you to create a lamp and genie would look something like this...

```rb
lamp = Lamp.create( wishes_remaining: 3 )
genie = Genie.create( name: "Genie" )
```